nlpsandboxclient/client.py

```diff
@@ -0,0 +1,64 @@
+"""NLP client object"""
```

Comment:
Start python files with `#!/usr/bin/env python3` for better portability (https://blogs.gnome.org/mcatanzaro/2018/02/16/on-python-shebangs/).